scripts/build_utils.sh

```diff
@@ -821,15 +821,34 @@ gen_debian_version() {
 # Flavor Builds support
 # - CEPH_EXTRA_RPMBUILD_ARGS is consumed by build_rpms()
 # - CEPH_EXTRA_CMAKE_ARGS is consumed by debian/rules and ceph.spec directly
-ceph_build_args_from_flavor() {
+ceph_build_args_from_flavor_and_dist() {
     local flavor=$1
     shift
+    local dist=$1
+    shift
 
     # shellcheck disable=SC2034
     case "${flavor}" in
     default)
         CEPH_EXTRA_RPMBUILD_ARGS="--with tcmalloc"
-        CEPH_EXTRA_CMAKE_ARGS+=" -DALLOCATOR=tcmalloc"
+        # we have to build quincy and up on bionic, because of some teuthology tests are
+        # still using ubuntu focal:
```

Comment:
"... because some teuthology tests are still using ubuntu bionic"

or

"... because of some teuthology tests that are still using ubuntu bionic"

Reply:
fixed.

Reply:
Not completely.  It still says ubuntu focal -- that should be ubuntu bionic.

Reply:
i am sorry, should be fixed now.